maintainers-circle/hosting.md

```diff
@@ -0,0 +1,87 @@
+# Hosting Maintainers Circle
+
+Hosts are responsible for planning and hosting the circle topic they are
+running. Below is guidance to help in this process. 
+
+## Planning
+
+### Curating content
+1. Check on the [README] and [issue backlog] to see what topics are in the queue
+ first. If no topics are left, initiate ideas with the sig in slack. Next step, 
+ ask directly in the #maintainers-circle channel as to what folks would like to 
+ discuss/learn.
+ 2. Add a short bio for the speakers
+
+Content should not be a sales pitch of any kind. It should match with the goals
+and intent of the group and have at least one 15 minute break out session to so
+that this isn't another talk. Networking and learning together is an important 
+piece to this. 
+The [structure] should follow the model on the README but can be tailored based
+on increasing breakout time/interactive reasons or really good content! 
+Try for one speaker that has info/science/data/research/teacher and another that
```

Comment:
s/that has /who has a 
s/and/background and 
s/another that/another who
s/that has experience/who has experience

Reply:
```suggestion
Try for one speaker who has a science/data/research/teacher background and another who has experience
```